src/assets/translations/en-US.json

```
@@ -1,832 +1,839 @@
-    "tooltip.Fee": "Transaction costs and payment method fees",
-    "tooltip.Other": "Transactions not in another category, usually an adjustment",
-    "tooltip.Payment": "Money received to your account from a sales transaction",
-    "tooltip.Refund": "Money sent back to a customer from a refunded transaction",
-    "tooltip.totalIncoming": "All money received into your account based on the selected filters",
-    "tooltip.totalOutgoing": "All fees, refunds, payouts, and other charges based on the selected filters",
-    "tooltip.Transfer": "Money moved between your account and another account",
-    "totalIncoming": "Total incoming",
-    "totalOutgoing": "Total outgoing",
-    "toValueShouldBeGreaterThanTheFromValue": "To value should be equal or greater than the From value",
-    "transactionDetails": "Transaction details",
-    "transactions": "Transactions",
-    "transactions.balances.label": "Account balances",
-    "transactions.copy.referenceId": "Copy reference ID",
-    "transactions.filters": "Transactions filters",
-    "transactions.pagination": "Transactions pagination",
-    "transactions.pagination.limitSelector.label": "Transactions per page",
-    "transactions.totals.label": "Transaction totals",
-    "transactionsOverviewTitle": "Transactions overview",
-    "transactionType": "Transaction type",
-    "transfer": "Transfers",
-    "tryDifferentSearchOrResetYourFiltersAndWeWillTryAgain": "Try a different search or reset your filters, and we’ll try again.",
-    "tryRefreshingThePageOrComeBackLater": "Try refreshing the page or come back later.",
-    "txAmount": "Amount",
-    "txType.ATM": "ATM",
-    "txType.Capital": "Capital",
-    "txType.Chargeback": "Chargeback",
-    "txType.Correction": "Correction",
-    "txType.Fee": "Fee",
-    "txType.Other": "Other",
-    "txType.Payment": "Payment",
-    "txType.Refund": "Refund",
-    "txType.Transfer": "Transfer",
-    "type": "Type",
-    "uploadedFile.remove": "Delete %{filename} file",
-    "uploadFile.browse": "Browse files",
-    "value": "Value",
-    "weCouldNotLoadThePayoutsOverview": "We couldn't load the payouts overview.",
-    "weCouldNotLoadTheReportsOverview": "We couldn't load the reports overview.",
-    "weCouldNotLoadTheTransactionsOverview": "We couldn't load the transactions overview.",
-    "weCouldNotLoadYourBalanceAccounts": "We couldn't load your balance accounts.",
-    "weCouldNotLoadYourPayouts": "We couldn't load your payouts.",
-    "weCouldNotLoadYourReports": "We couldn't load your reports.",
-    "weCouldNotLoadYourTransactions": "We couldn't load your transactions."
+    "capital.common.actions.contactSupport": "Contact support",
+    "capital.common.actions.goBack": "Go back",
+    "capital.common.errors.unsupportedRegion": "Business financing isn’t available in your region yet, but check back here for an offer.",
+    "capital.common.errors.unsupportedRegion.title": "Stay tuned!",
```

In this case it's clear that unsupportedRegion is the group. Makes sense?

```suggestion
    "capital.common.errors.unsupportedRegion.description": "Business financing isn’t available in your region yet, but check back here for an offer.",
    "capital.common.errors.unsupportedRegion.title": "Stay tuned!",
```